rpcd/playbooks/roles/filebeat/templates/filebeat.yml.j2

```diff
@@ -1,2 +1,42 @@
 # {{ ansible_managed }}
-{{ filebeat_config_overrides | to_yaml }}
+filebeat:
+  prospectors:
+{% for service in filebeat_logging_paths %}
+    - paths: {{ service.paths }}
+{% if service.document_type is defined %}
+      document_type: {{ service.document_type }}
+{% endif %}
+      encoding: {{ service.encoding |default('utf-8') }}
+      input_type: log
```

Comment:
I like to be explicit rather than implicit @evrardjp. We don't have to specify this but the filebeat documentation doesn't describe what the default value for this might be.

Further, filebeat 5.0 is the next version which is breaking things with the underlying beats implementation but shouldn't affect filebeat (according to their developers in their IRC channel). I trust them, but want to make sure there's no implicit default that changes with this.